config/production.json

```diff
@@ -94,6 +94,7 @@
 		"plans/pro-plan": false,
 		"plans/starter-plan": false,
 		"post-editor/checkout-overlay": true,
+		"pre-cancellation-modal": false,
```

Comment:
Let's also add this to wpcalypso.json and stage.json as a complete set :)

Reply:
The flag pre-cancellation-modal has been added to `wpcalypso.json` and `stage.json`